src/org/labkey/targetedms/parser/skyaudit/DatabaseUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.labkey.targetedms.parser.skyaudit;
+
+import org.labkey.api.data.BaseSelector;
+import org.labkey.api.data.SQLFragment;
+import org.labkey.api.data.SqlExecutor;
+import org.labkey.targetedms.TargetedMSSchema;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DatabaseUtil
+{
+
+    public static Object retrieveSimpleType(SQLFragment pQuery){
```

Comment:
Consider using SqlSelector instead of SqlExecutor, where you should be able to call new SqlSelector().getObject(Integer.class) and SqlSelector.getMap()